src/css/var/rmargin.js

```diff
@@ -1,3 +1,3 @@
-define(function() {
-	return (/^margin/);
-});
+define( function() {
+	return ( /^margin/ );
```

Comment:
Don't know, but that is not a subject of this PR, too much already is squashed in there :-)